pysaliency/dataset_config.py

```diff
@@ -52,6 +54,115 @@ def apply_dataset_filter_config(stimuli, fixations, filter_config):
     return filter_fn(stimuli, fixations, **filter_config['parameters'])
 
 
+def filter_scanpaths_by_attribute(scanpaths: FixationTrains, whitelist: dict=None, blacklist: dict=None):
```

Comment:
The filter functions themselves should go into `filter_datasets.py`. The present file, `dataset_config` should only use them to extend `apply_dataset_filter_config` (which is still missing from this PR).